ios/DdLogs.swift

```diff
@@ -9,11 +9,7 @@ import DatadogSDKBridge
 
 @objc(DdLogs)
 class RNDdLogs: NSObject {
-    @objc(requiresMainQueueSetup)
-    static func requiresMainQueueSetup() -> Bool {
-        return false
-    }
```

Comment:
absence of this method was causing warnings in debug console.
i'm guessing it still gives warnings as RN version isn't changed.
did you run it in iOS simulator?

Reply:
Again, this class is auto generated, if this change is important, we should make the change in the generator classes. 
And yes, I ran it in the iOS simulator, and it all worked.

Reply:
it works, it doesn't give an error; it gives a warning

Reply:
does that change require a new PR in `mobile-bridge` repo?